packages/client/public/locales/en/home.json

```
@@ -0,0 +1,79 @@
+    "chooseOpponent": "Choose your opponent",
+    "apprentice": "Apprentice",
+    "apprenticeDesc": "Plays loosely, makes mistakes.",
+    "gentleman": "Gentleman",
+    "gentlemanDesc": "Plays smart, occasional blunders.",
+    "highRoller": "High Roller",
+    "highRollerDesc": "Ruthless. Calculates every card."
+  },
+  "custom": {
+    "title": "Custom Table",
+    "create": "Create",
+    "join": "Join",
```

Biome (2.4.15)

[error] The key join was already declared.

(lint/suspicious/noDuplicateObjectKeys)